components/metricsView/cards/ForwardsRatingNode.component.tsx

```diff
@@ -0,0 +1,301 @@
+  const dataOneDay = [
+    {
+      id: "success",
+      label: "success",
+      value: metrics.forwards_rating.one_day.success,
+      color: "hsl(146, 55%, 54%)",
+    },
+    {
+      id: "failure",
+      label: "failure",
+      value: metrics.forwards_rating.one_day.failure,
+      color: "hsl(0, 100%, 50%)",
+    },
+    {
+      id: "internal_failure",
+      label: "internal_failure",
+      value: metrics.forwards_rating.one_day.internal_failure,
+      color: "hsl(340, 70%, 50%)",
+    },
+  ];
+
+  const hello: any = [];
+
+  const dataTenDays = [
+    {
+      id: "success",
+      label: "success",
+      value: metrics.forwards_rating.ten_days.success,
+      color: "hsl(146, 55%, 54%)",
+    },
+    {
+      id: "failure",
+      label: "failure",
+      value: metrics.forwards_rating.ten_days.failure,
+      color: "hsl(0, 100%, 50%)",
+    },
+    {
+      id: "internal_failure",
+      label: "internal_failure",
+      value: metrics.forwards_rating.ten_days.internal_failure,
+      color: "hsl(340, 70%, 50%)",
+    },
+  ];
+
+  const dataThirtyDays = [
+    {
+      id: "success",
+      label: "success",
+      value: metrics.forwards_rating.thirty_days.success,
+      color: "hsl(146, 55%, 54%)",
+    },
+    {
+      id: "failure",
+      label: "failure",
+      value: metrics.forwards_rating.thirty_days.failure,
+      color: "hsl(0, 100%, 50%)",
+    },
+    {
+      id: "internal_failure",
+      label: "internal_failure",
+      value: metrics.forwards_rating.thirty_days.internal_failure,
+      color: "hsl(340, 70%, 50%)",
+    },
+  ];
+
+  const dataSixMonths = [
+    {
+      id: "success",
+      label: "success",
+      value: metrics.forwards_rating.six_months.success,
+      color: "hsl(146, 55%, 54%)",
+    },
+    {
+      id: "failure",
+      label: "failure",
+      value: metrics.forwards_rating.six_months.failure,
+      color: "hsl(0, 100%, 50%)",
+    },
+    {
+      id: "internal_failure",
+      label: "internal_failure",
+      value: metrics.forwards_rating.six_months.internal_failure,
+      color: "hsl(340, 70%, 50%)",
+    },
+  ];
+
+  const colors: any = {
+    success: "#32A966",
+    failure: "#FD706B",
+    internal_failure: "#FFB347",
+  };
+  const getColor = function (bar: any) {
+    return colors[bar.id];
+  };
+
+  return (
+    <Card className={styles.cardContainer}>
+      <CardHeader
+        avatar={
+          <Avatar aria-label="recipe">
+            <Timelapse />
+          </Avatar>
+        }
+        title={`Node ${nodeInfo.alias} forwards rating`.toUpperCase()}
+      />
+      <CardContent>
+        <Grid
+          container
+          direction="row"
+          justifyContent="left"
+          item
+          xs={12}
+          xl={12}
+          sm={12}
+          alignItems="left"
+        >
+          <Grid item xs={12} xl={6} sm={6}>
+            <div className={styles.containerPie}>
+              <h4 className={styles.pieText}>One Day</h4>
```

Comment:
```suggestion
            <div className={styles.containerPie}>
              <h4 className={styles.pieText}>One Day</h4>
```

Can use material components here? Like `Container` for div? and something else for `h4`? you can find it in the components doc page of miu